package P_20211222.exception01;

public class Exception02P348 {
	public static void main(String[] args) {
		//예외발생 케이스 2
		int a = 1;
		int b = 0;
		
		System.out.println(a/b);
	}
}
